resources/ci/docker/Dockerfile.x64Linux3gcc4.8.2

```diff
@@ -0,0 +1,48 @@
+# (c) 2024 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+# RTI grants Licensee a license to use, modify, compile, and create derivative
+# works of the Software.  Licensee has the right to distribute object form only
+# for use with RTI products.  The Software is provided "as is", with no warranty
+# of any type, including any warranty for fitness for any purpose. RTI is under
+# no obligation to maintain or support the Software.  RTI shall not be liable
+# for any incidental or consequential damages arising out of the use or
+# inability to use the software.
+
+from ubuntu:14.04
```

Comment:
Just a question: should you use CentOS 7 for this?

Reply:
this is not in use, we will use our docker containers in the next iteration